Arrowgene.Ddon.GameServer/Characters/CharacterManager.cs

```diff
@@ -59,6 +59,14 @@ public Character SelectCharacter(uint characterId, DbConnection? connectionIn =
                 return null;
             }
 
+            character.EpitaphRoadState.UnlockedContent = _Server.Database.GetEpitaphRoadUnlocks(character.CharacterId);
```

Comment:
I'm not sure what you mean by this. Every character needs the details populated. Old logic should be moved into the character manager instead of the way it was originally designed so we can move the transaction logic into these functions.

Reply:
This function is called when you log in.

Reply:
I see it got used in some of these pawn ones. Maybe we should make a lighter ones for those? Problem is pawns need the character inventory and all that other unpleasant stuff.